pkg/arvo/sys/hoon.hoon

```diff
@@ -14132,14 +14133,14 @@
       ::
       ::  Here's an example of what a hint looks like.
       ::
-      ::      ~>(%mean.[%leaf "need"] !!)
+      ::      ~>(%mean.'need' !!)
       ::
       ::  The actual form that we're printing here looks something like this:
       ::
-      ::      %mean.[%leaf "need"]
+      ::      %mean.'need'
       ::
-      ::  XX I'm not sure if the `[%leaf "need"]` bit represents a literal
-      ::  AST fragment or an expression that evaluates to `[%leaf "need"]. I'm
+      ::  XX I'm not sure if the `'neeed` bit represents a literal
+      ::  AST fragment or an expression that evaluates to `'need'`. I'm
```

Comment:
This comment is explaining an approach taken in the new pretty-printer. I think you should leave it as is, maybe adding a comment noting that these hint forms were wrong.